src/i18n.js

```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue'
```

Comment:
This should be enough, but test it the translation still works `
import VueI18n from 'vue-i18n'
import Vue from 'vue'
import enLocaleFile from '@/assets/lang/en-US.json'

export default function ({ app }) {
  Vue.use(VueI18n)
  app.i18n = new VueI18n({
    messages: {
      en: enLocaleFile
    },
    locale: 'en',
    fallbackLocale: 'en',
    silentTranslationWarn: true
  })
}`

Reply:
<img width="1440" alt="Screenshot 2021-08-17 at 12 01 30" src="https://user-images.githubusercontent.com/89005333/129696866-2a1f32a4-443e-4ea4-b1d7-b25679901fa5.png">